geekodoc/tests/run-tests.sh

```diff
@@ -139,17 +136,29 @@ function test_files() {
     local re='^[0-9]+$'
 
     for xmlfile in $DIR/*.xml; do
-       validator $SCHEMA $xmlfile
-       result=$?
-       resultstr=$(test_check $GOOD_OR_BAD $result)
-       count_errors $GOOD_OR_BAD $result
-       loginfo "Validating '$xmlfile'... $resultstr"
+       loginfo -n "Validating '$xmlfile'... "
+       wellformedness=$(xmllint --noout --noent $xmlfile 2>&1)
```

Comment:
A wellformedness check is really cool! I haven't thought about that! 👍

Wouldn't it be "better" to check for the return code and *not* the output? Something like:

```bash
xmllint --noout --noent $xmlfile 2>&1
if [ $? -ne 0 ]; then
# ....
```

Reply:
Again, the way I am doing it, means I write the validation result into a variable, so I can't use the return code.